etl/src/main/java/org/pih/hivmigration/etl/sql/Migrator.java

```diff
@@ -87,6 +87,7 @@ public void main() {
             if (shouldRevert || shouldRevertOnly) {
                 revert(new MedpickupsMigrator());
                 revert(new VitalsMigrator());
+                revert(new ReturnVisitDateMigrator());
```

Comment:
@cioan - if we create a Migrator for every Obs in the system, it's going to get unmanageable.  We should be a little more thoughtful about how we create migrators for specific areas of data.  Let's think through a bit more where "Return Visit Date" falls.  I could see options like:

* TreatmentPlanMigrator - migrates everything in the last few sections of the intake/followup forms
* ExamExtraMigrator - migrates everything out of the Exam Extra table that is needed

For now, you can still just add the single data point migration, but let's give ourselves some room to expand an existing migration to handle more data in the domain, rather than have to create hundreds of Migrator classes.